clients/java/src/main/java/io/camunda/client/api/command/CompleteJobCommandStep1.java

```diff
@@ -89,5 +90,74 @@ interface CompleteJobCommandStep2 extends FinalCommandStep<CompleteJobResponse>
      * @return the builder for this command.
      */
     CompleteJobCommandStep2 denied(boolean denied);
+
+    /**
+     * Attributes that were corrected by the worker. The below attributes can be corrected and
+     * additional attributes will be ignored: `assignee` - reset by providing an empty String
```

Comment:
> additional attributes will be ignored

🔧 Since all correctable attributes are already defined in `CompleteJobCommandStep3`, this note may not be relevant.